test/integration/apiserver/flowcontrol/fight_test.go

```diff
@@ -74,12 +74,13 @@ type fightTest struct {
 
 func newFightTest(t *testing.T, loopbackConfig *rest.Config, teamSize int) *fightTest {
 	now := time.Now()
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
In unit tests use `_, ctx := ktesting.NewTestContext(t)` to get per-test output.

Reply:
done